client/plots/survival.js

```diff
@@ -1061,75 +1047,43 @@ function setInteractivity(self) {
 			//callback: d => {}
 			setInput: holder => {
 				const label = holder.append('label')
-				label
-					.append('span')
-					.style('vertical-align', 'middle')
-					.style('line-height', '25px')
-					.html('Edit color ')
+				label.append('span').style('vertical-align', 'middle').style('line-height', '25px').html('Edit color ')
 				const input = label
 					.append('input')
 					.attr('type', 'color')
 					.attr('value', self.term2toColor[d.seriesId].hex)
 					.style('vertical-align', 'top')
 					.on('change', () => self.adjustColor(input.property('value'), d))
-
-				holder
-					.append('span')
-					.style('vertical-align', 'middle')
-					.style('line-height', '25px')
-					.html(' OR ')
-
-				holder
-					.append('button')
-					.style('margin-left', '5px')
-					.style('background-color', self.term2toColor[d.seriesId].rgb.darker())
-					.style('vertical-align', 'top')
-					.html('darken')
-					.on('click', () => self.adjustColor(input.property('value'), d, 'darker'))
-
-				holder
-					.append('button')
-					.style('margin-left', '5px')
-					.style('background-color', self.term2toColor[d.seriesId].rgb.brighter())
-					.style('vertical-align', 'top')
-					.html('brighten')
-					.on('click', () => self.adjustColor(input.property('value'), d, 'brighter'))
 			}
 		})
 
 		if (!options.length) return
-		self.activeMenu = true
-		self.app.tip.clear()
-		self.app.tip.d.append('div').html(header)
-		self.app.tip.d
+		const menu = new Menu()
```

Comment:
this creates one new menu instance at every click. should create a persistent one at `self.legendMenu` and reuse it